.travis.yml

```diff
@@ -2,7 +2,7 @@ language: python
 
 python:
     - 2.7
-    - 3.6
```

Comment:
If you have dropped python 2.7 then no point keep testing it. Remove it from the line above from here, too, and from below in the allowed failures, too.

Reply:
I left it just because it might be useful to keep track of the features that are still working on Python 2.7 over time. Do you think it's useless?

Reply:
Sure. It's certainly not the typical approach, usually when people say they drop support it means all hands off for those versions. But then I have no idea at all from your user base, it may be much more python2 bounded than for other cases.